### ISSUE TITLE
is there a way to load json cookies in nodriver? like in uc

### ISSUE THREAD

**Issue by rex1de** [ultrafunkamsterdam/undetected-chromedriver#1838]
is there a way to load json cookies in nodriver? like in uc

**Comment by usr741852** [comment]
yes, but there is a bug in the code that you need to fix in order for the cookies to work, you can see the issue: https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1816


```
await browser.cookies.set_all(cookies)
```

is how you set the cookies

```
cookies = []
c = CookieParam(name='n',value='v',path='/',domain='google.com')
cookies.append(c)

```

cookies aren't JSON, you need to pass in the above list of CookieParam class

**Comment by Metug** [comment]
> cookies aren't JSON, you need to pass in the above list of CookieParam class

What is CookieParam?
Thanks.
